examples/widget-context-menu-with-3rd-party-data/package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "datadog-app-example-widget-context-menu-with-3rd-party-data",
+    "version": "0.0.0",
+    "private": true,
+    "dependencies": {
+        "@datadog/ui-extensions-react": "0.31.0",
+        "@datadog/ui-extensions-sdk": "0.31.0",
+        "react": "18.0.0",
+        "react-dom": "18.0.0"
+    },
+    "devDependencies": {
+        "@types/node": "^14.14.14",
+        "@types/react": "^17.0.0",
+        "@types/react-dom": "^17.0.0",
+        "react-scripts": "5.0.0",
+        "typescript": "^4.5.4"
+    },
+    "scripts": {
+        "build": "react-scripts build",
+        "eject": "react-scripts eject",
+        "format:check": "prettier --check .",
+        "format:fix": "prettier --write .",
+        "lint:check": "eslint .",
+        "lint:fix": "yarn run lint:check --fix",
+        "lint": "yarn run lint:check && yarn run format:check",
+        "prepare": "exit 0",
```

Comment:
**tl;dr;** This is so that the [top-level `"prepare"` script](https://github.com/DataDog/apps/blob/3e5e95858e6b3130a4ae6b1ba5389aed187121fd/package.json#L17) works. Because we say `yarn workspaces run prepare`, all of the workspaces need to have a `"prepare"` script, even if they do nothing except exit successfully.

---

The more in-depth answer is that there's a combination of things that cause this to happen:
- We're using yarn classic workspaces.
- We're using `yarn workspaces run`.
- We're developing in TypeScript.
- We're depending on other workspaces within this repo.

All of these things come together to mean that we need to figure out some way to build workspaces like `@datadog/ui-extensions-sdk` before attempting to do anything in a workspace like this example. The way we decided at the time was to lean on the `"prepare"` script since it runs before most other scripts in the lifecycle. Nothing requiring this to be the only way, and we have a Jira Issue to try and improve the situation: https://datadoghq.atlassian.net/browse/UIAPPS-135.

Reply:
Ah I see. Thanks for the background. I wasn't aware of the necessity of the `prepare` script.